openassessment/xblock/staff_area_mixin.py

```diff
@@ -322,6 +323,7 @@ def get_student_submission_context(self, student_username, submission):
             'submission': create_submission_dict(submission, self.prompts) if submission else None,
             'rubric_criteria': copy.deepcopy(self.rubric_criteria_with_labels),
             'student_username': student_username,
+            'time_zone': get_current_time_zone(self.runtime.service(self, 'user')),  # localize for staff user
```

Comment:
It's expected that a staff users perusing user responses would see times in their own (the staff user's) timezone, correct?